JustSaying/Messaging/IMessagePublisher.cs

```diff
@@ -1,13 +1,11 @@
-﻿using System.Threading;
+using System.Threading;
 using System.Threading.Tasks;
 using JustSaying.Models;
 
 namespace JustSaying.Messaging
 {
     public interface IMessagePublisher
     {
-        Task PublishAsync(Message message);
-
-        Task PublishAsync(Message message, CancellationToken cancellationToken);
+        Task PublishAsync(Message message, PublishMetadata metadata, CancellationToken cancellationToken);
```

Comment:
Same. I imagine one set would cover all the cases where I'm making this comment.